source/codegen/templates/service_helpers.mako

```diff
@@ -112,20 +174,21 @@ ${set_response_values(output_parameters=output_parameters)}\
   input_parameters = [p for p in parameters if common_helpers.is_input_parameter(p)]
 %>\
 % for parameter in input_parameters:
-${initialize_input_param(function_name, parameter)}
+${initialize_input_param(function_name, parameter)}\
```

Comment:
Note: basically I inverted the placement of the newline-eating `\` so that no newlines are placed by the for loop but newlines *are* placed by any `initialize_whatever_param` below.

This allows my new `## pass` to no-op-generate instead of generating a newline.